src/dgipy/dgidb.py

```diff
@@ -220,6 +220,75 @@ def get_drug_applications(
     return result
 
 
+def get_clinical_trials(
```

Comment:
This should probably be in a separate module since it's not really about querying DGIdb, but rather about incorporating an external service

Reply:
Would that be something to redo here for this PR? Or could we get this in and then separate it into an external module after all the functionality is deemed suitable.

Asking because we another method (_openfda_data) also grabs from an external service. Might make sense to export them both together into some kind of "clinical" module?

Reply:
We can make an additional issue along the lines of "move non-dgidb methods out of the dgidb module"